how-to/workspace-platform-starter/client/src/framework/launch.ts

```diff
@@ -372,12 +382,99 @@ async function launchWindow(windowApp: PlatformApp): Promise<PlatformAppIdentifi
 
 	if (!windowExists) {
 		try {
+			const appLaunchPreference = objectClone(windowApp.launchPreference);
+			const appLaunchPreferenceOptions = objectClone(
+				windowApp.launchPreference?.options
+			) as WindowLaunchOptions;
+
+			// validate passed launch preferences
+			if (
+				appLaunchPreference?.options?.type === "window" &&
+				Array.isArray(appLaunchPreferenceOptions?.updatable) &&
+				appLaunchPreferenceOptions.updatable.length > 0
+			) {
+				console.log(appLaunchPreference.options.updatable);
+				for (const option of appLaunchPreferenceOptions.updatable) {
+					if (option.name === "BOUNDS") {
+						appLaunchPreference.bounds = launchPreference?.bounds;
+					}
+					if (option.name === "CENTERED") {
```

Comment:
change to `else if` for all the option.name or a `switch`

Reply:
went for else if as there are some additional checks.